### PR TITLE
I Added LoggerHometask

### PR DESCRIPTION
**Pull request by Vitaliiitk**

### CODE REVIEW

File: LoggerHomeTask/LoggerHomeTask/Actions.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace LoggerHomeTask
+{
+    public class Actions
+    {
+
+        public Result Method1()
```
Comment (petrochernykh) [inline LoggerHomeTask/LoggerHomeTask/Actions.cs:13]: You should use more descriptive naming

File: LoggerHomeTask/LoggerHomeTask/Actions.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        }
+
+    }
+    public class Result
+    {
+        public string Type { get; set; }
+        public bool Status { get; }
+        public string ErrorMessage { get; }
+
+        public Result(string type)
```
Comment (petrochernykh) [inline LoggerHomeTask/LoggerHomeTask/Actions.cs:47]: Looks good. Also you can use Enum to send type into constructor.

File: LoggerHomeTask/LoggerHomeTask/Actions.cs (hunk 1 above)
Comment (Vitaliiitk) [reply LoggerHomeTask/LoggerHomeTask/Actions.cs:13]: Ok, thank you for comment.

File: LoggerHomeTask/LoggerHomeTask/Actions.cs (hunk 2 above)
Comment (Vitaliiitk) [reply LoggerHomeTask/LoggerHomeTask/Actions.cs:47]: Enum was not obvious for me, I remember you told me to do that in Telegram. If I have any time I will try to implement Enum. Right now another HW is waiting for me. Overall, thank for advice.

**Pull request merged**